README.md

```diff
@@ -1 +1,1146 @@
+```
+
+Create a file `prometheus.yml` inside the `/tmp/` location. Add the below configurations to the `prometheus.yml` file.
+
+```
+   global:
+     scrape_interval:     15s
+     evaluation_interval: 15s
+
+   scrape_configs:
+     - job_name: prometheus
+       static_configs:
+         - targets: ['172.17.0.1:9797']
+```
+
+> **NOTE**: Replace `172.17.0.1` if your local Docker IP differs from `172.17.0.1`
+   
+Run the Prometheus docker image using the following command
+
+```
+   $ docker run -p 19090:9090 -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+   prom/prometheus
+```
+   
+You can access Prometheus at the following URL.
+
+```
+   http://localhost:19090/
+```
+
+> **NOTE**: Ballerina will by default have the following metrics for HTTP server connector. You can enter the following expression in Prometheus UI.
+>    -  http_requests_total
+>    -  http_response_time
+
+### Logging
+Ballerina has a log package for logging to the console. You can import `ballerina/log` package and start logging. The following section describes how to search, analyze, and visualize logs in real time using Elastic Stack.
+
+Start the Ballerina service with the following command from `eip-message-transformation/guide`
+
+```
+   $ nohup ballerina run message_transformation/ &>> ballerina.log&
+```
+
+> **NOTE**: This writes the console log to the `ballerina.log` file in the `eip-message-transformation/guide` directory.
+
+Start Elasticsearch using the following command.
+
+```
+   $ docker run -p 9200:9200 -p 9300:9300 -it -h elasticsearch --name \
+   elasticsearch docker.elastic.co/elasticsearch/elasticsearch:6.2.2 
+```
+
+> **NOTE**: Linux users might need to run `sudo sysctl -w vm.max_map_count=262144` to increase `vm.max_map_count`.
+   
+Start Kibana plugin for data visualization with Elasticsearch.
+
+```
+   $ docker run -p 5601:5601 -h kibana --name kibana --link \
+   elasticsearch:elasticsearch docker.elastic.co/kibana/kibana:6.2.2     
+```
+
+Configure logstash to format the Ballerina logs.
+
+i) Create a file named `logstash.conf` with the following content.
+
+```
+input {  
+ beats{ 
+     port => 5044 
+ }  
+}
+
+filter {  
+ grok{  
+     match => { 
+     "message" => "%{TIMESTAMP_ISO8601:date}%{SPACE}%{WORD:logLevel}%{SPACE}
+     \[%{GREEDYDATA:package}\]%{SPACE}\-%{SPACE}%{GREEDYDATA:logMessage}"
+     }  
+ }  
+}   
+
+output {  
+ elasticsearch{  
+     hosts => "elasticsearch:9200"  
+     index => "store"  
+     document_type => "store_logs"  
+ }  
+}  
+```
+
+ii) Save the above `logstash.conf` inside a directory named as `{SAMPLE_ROOT}\pipeline`.
+     
+iii) Start the logstash container, replace the `{SAMPLE_ROOT}` with your directory name.
+     
+```
+$ docker run -h logstash --name logstash --link elasticsearch:elasticsearch \
+-it --rm -v ~/{SAMPLE_ROOT}/pipeline:/usr/share/logstash/pipeline/ \
+-p 5044:5044 docker.elastic.co/logstash/logstash:6.2.2
+```
+  
+Configure filebeat to ship the Ballerina logs.
```

Comment:
This seems like a new section so let's add some markdown to indicate that.

Reply:
Done